styles/components/_past-clients.scss

```diff
@@ -1,5 +1,6 @@
 .cmp-past-clients {
   width: 100%;
+  margin-top: space("xl");
```

Comment:
For FunFacts, PastClients, and PersonalBio we could similarly have the largest, Figma margin here, too.
```css
  @media (min-width: $lg-screen-width) {
    margin-top: space("3xl");
  }
```